trino-rest-github/src/main/java/pl/net/was/rest/github/Sync.java

```diff
@@ -1852,12 +1855,23 @@ private static int getEmptyLimit()
         return Integer.parseInt(limit);
     }
 
+    private static long getQueryRetryDelayMillis()
+    {
+        String queryRetryDelay = System.getenv("QUERY_RETRY_DELAY_MS");
+        if (queryRetryDelay == null) {
+            return 500L;
```

Comment:
By default there should be no delay. For small repos it's not needed

Reply:
Adde default value of 0